src/amo/components/RatingsByStar/index.js

```diff
@@ -143,13 +143,19 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
                   )}
                   <IconStar selected />
                 </div>
-                <div className="RatingsByStar-barContainer">
-                  <div className="RatingsByStar-bar RatingsByStar-barFrame">
-                    {starCount !== undefined
-                      ? this.renderBarValue(starCount)
-                      : null}
-                  </div>
-                </div>
+                {loading ? (
+                  <div />
```

Comment:
Just displaying a `<div />` when loading is not quite as nice as it appeared before this change. I think we should show the `RatingsByStar-barContainer` div in either case, and just not call `createLink` if `loading` is true.  Can you give that a try @mirefly?